models/commons/src/high-precision-money/high-precision-money-draft/presets/index.ts

```diff
@@ -0,0 +1,3 @@
+const presets = {};
+
+export default presets;
```

Comment:
there is a need of this ? we are not using the presets in this money type ? .

Reply:
> there is a need of this ? we are not using the presets in this money type ? .

The presets object is also present in the rest of the models that don't provide any preset. I have no info on why this is the current approach of the package, I assume it is because of consistency for the user of the models.